numpy/_core/tests/test_limited_api.py

```diff
@@ -67,6 +67,12 @@ def install_temp(tmpdir_factory):
         "and Py_REF_DEBUG"
     ),
 )
+@pytest.mark.xfail(
+    sysconfig.get_config_var("Py_GIL_DISABLED"),
+    reason=(
+        "Py_LIMITED_API is incompatible with the Python 3.6 limited API"
```

Comment:
This reason seems self-contradictory - you probably meant to start with `Py_GIL_DISABLED` instead? I'm not sure the 3.6 is helpful either, I think that the limited API is not supported by the free-threaded CPython build at all yet.